vtex-apps/checkout-cart
# Show toast when removing an item from cart

#### What problem is this solving?

This makes a toast show up containing just the text "Item ${name} was removed from the cart" when the user removes an item from the cart. This is one step towards implementing the undo-remove feature.

#### How should this be manually tested?

Add [some item](https://toast--vtexgame1.myvtex.com/camisa-vasco-preta/p) to your cart, then go to `/cart`. Remove the item and verify the toast appears.

#### Checklist/Reminders

- [x] [Linked this PR to a Clubhouse story (if applicable)](https://app.clubhouse.io/vtex/story/23743/mostrar-toast-no-carrinho-após-exclusão-de-produto-com-opção-para-desfazer-ação).

#### Screenshots or example usage

#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements 

#### Notes
